kyo-combinators/shared/src/main/scala/kyo/Constructors.scala

```diff
@@ -243,11 +231,8 @@ extension (kyoObject: Kyo.type)
       * @return
       *   An effect that logs the message and error to the console
       */
-    inline def logInfo[S, S1](
-        message: => String < S,
-        err: => Throwable < S1
-    ): Unit < (S & S1 & IO) =
-        message.map(m => err.map(e => Log.info(m, e)))
+    inline def logInfo(message: String, err: Throwable): Unit < IO =
```

Comment:
I believe all Logging functions should accept functions by name.

Reply:
They were missing `inline` as well